src/Components/Reports/Common/ExecutiveSecondPage.js

```diff
@@ -1,97 +1,58 @@
-            iconScale: 0.008,
-            fillColor: '#2B9AF3'
-        }
-    ];
+    const { current, suggested, historical } =  data?.instance_types_highlights;  /* eslint-disable-line camelcase */
+    const { stale_count: staleCount } = data?.meta;
 
-    const conditionsDescription = [
-        {
-            title: 'CPU pressure',
-            description: 'CPU registered peaks higher than 20% over several one-minute time periods'
-        },
-        {
-            title: 'Disk I/O  pressure',
-            description: 'Disk I/O registered peaks higher than 20% over several one-minute time periods'
-        },
-        {
-            title: 'RAM pressure',
-            description: 'RAM registered peaks higher than 20% over several one-minute time periods'
-        }
-    ];
+    const currentInstancesDetails = {
+        heading: 'Current instance types',
+        description: 'This is your current state, representing the instances that are already reporting data.',
+        data: current
+    };
 
-    return <Fragment key="second-page">
-        <Text style={styles.execHeading}>Description of states</Text>
-        {
-            stateDescription.map(
-                (state) => <DescriptionList
-                    key={state.title}
-                    title={state.title}
-                    description={state.description}
-                    iconPath={state.iconPath}
-                    iconScale={state.iconScale}
-                    fillColor={state.fillColor} />)
-        }
+    const suggestedInstancesDetails = {
+        heading: 'Most suggested instance types (yesterday)',
+        description: 'Yesterday we identified these instance types',
+        data: suggested
+    };
 
-        <Text style={styles.execHeading}>Description of conditions</Text>
-        {
-            conditionsDescription.map(
-                (condition) => <DescriptionList
-                    key={condition.title}
-                    title={condition.title}
-                    description={condition.description}/>)
-        }
+    const historicalInstancesDetails = {
+        heading: 'Most suggested instance types (45 days)',
+        description: 'In the last 45 days we suggested you these instances # of times. ',
+        staleDescription: 'Report includes instances running on systems service identified as stale.',
+        data: historical
+    };
+
+    return <Fragment key="third-page">
+        <InstancesTable
+            id={'current_instance_types'}
+            instanceDetails={currentInstancesDetails.data}
+            heading={currentInstancesDetails.heading}
+            description={currentInstancesDetails.description}
+
+        />
+
+        <InstancesTable
+            id='suggested_instance_types'
+            instanceDetails={suggestedInstancesDetails.data}
+            heading={suggestedInstancesDetails.heading}
+            description={suggestedInstancesDetails.description}
+        />
```

Comment:
Will it be possible to extract out this into small functional component and reuse it?

Reply:
@kgaikwad I had extracted the instanceTable logic into functional component and reused it here the same. Do you mean creating one more component which will have these 3 tables?

Reply:
@kgaikwad I had extracted the instanceTable logic into functional component and reused it here the same. Do you mean creating one more component which will have these 3 tables?

Reply:
will convert the instanceDetails from object to array and iterate over it by using the InstanceTable component.